src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Models/AsAzureClientSession.cs

```diff
@@ -0,0 +1,148 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security;
+using Microsoft.Azure.Commands.AnalysisServices.Dataplane.Models;
+using Enumerable = System.Linq.Enumerable;
+
+namespace Microsoft.Azure.Commands.AnalysisServices.Dataplane
+{
+    /// <summary>
+    /// Represents current AS Azure session.
+    /// </summary>
+    public class AsAzureClientSession
+    {
+        public const string RestartEndpointPathFormat = "/webapi/servers/{0}/restart?api-version=2016-10-01";
+        public const string AsAzureClientId = "cf710c6e-dfcc-4fa8-a093-d47294e44c66";
+        public static readonly Uri RedirectUri = new Uri("urn:ietf:wg:oauth:2.0:oob");
+        public static string DefaultRolloutEnvironmentKey = "asazure.windows.net";
+
+        public static Dictionary<string, AsAzureAuthInfo> AsAzureRolloutEnvironmentMapping = new Dictionary<string, AsAzureAuthInfo>()
+            {
+             { "asazure-int.windows.net", new AsAzureAuthInfo()
```

Comment:
You should not include internal endpoints here.  No customers will ever use this.  If using the internal endpoints is needed, you should have a mechanism for adding a new environment